libs/util/git/src/git-client.service.ts

```diff
@@ -256,6 +260,8 @@ export class GitClientService {
     baseBranch: string;
     repositoryGroupName?: string;
     pullRequestTitle?: string;
+    resourceName?: string;
+    resourceUrl?: string;
```

Comment:
It would make more sense to pass the `pullRequestBody` already manipulated to this function as in case that changes, the logic won't be flexible enough.